ant/src/org/jetbrains/kotlin/ant/Kotlin2JvmTask.kt

```diff
@@ -73,4 +79,48 @@ class Kotlin2JvmTask : KotlinCompilerBaseTask() {
         if (noReflect) args.add("-no-reflect")
         if (includeRuntime) args.add("-include-runtime")
     }
+
+    override fun execute() {
+        if (!fork)
+            super.execute()
+        else {
+            exec()
+        }
+    }
+
+    private fun exec() {
+        val javaHome = System.getProperty("java.home")
+        val javaBin = javaHome + separator + "bin" + separator + "java"
+        val classpath = System.getProperty("java.class.path")
+        var redirector = Redirector(this)
```

Comment:
Please fix the warning here and on `exitCode` below ("Variable is never modified and can be declared immutable using 'val'")

Reply:
Warnings have been fixed.